projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag/default-events.spec.ts

```diff
@@ -0,0 +1,489 @@
+import * as anonymousConsents from '../../../../helpers/anonymous-consents';
+import * as checkoutFlowPersistentUser from '../../../../helpers/checkout-as-persistent-user';
+import * as checkoutFlow from '../../../../helpers/checkout-flow';
+import * as loginHelper from '../../../../helpers/login';
+import { navigation } from '../../../../helpers/navigation';
+import * as productSearch from '../../../../helpers/product-search';
+import {
+  createProductQuery,
+  QUERY_ALIAS,
+} from '../../../../helpers/product-search';
+import {
+  cdsHelper,
+  strategyRequestAlias,
+} from '../../../../helpers/vendor/cds/cds';
+import { profileTagHelper } from '../../../../helpers/vendor/cds/profile-tag';
+
+describe('Profile-tag events', () => {
```

Comment:
Are those changes related to this PR?

Reply:
I can see in the PR to 4.3.x we have only changed the path of this file
https://github.com/SAP/spartacus/pull/14753/files#diff-f0e7debfd7f5766491e0bcf6e6e36a7ebaf209e2cc19a22c8002657ab9d2b4c1
<img width="1576" alt="image" src="https://user-images.githubusercontent.com/4001059/154490477-730b9b86-4b2c-418f-9545-f321aaff8cc1.png">

Reply:
@Platonn

What happened here was that the name of the file was changed in the `develop` branch and in my set of changes and the merge process added the file that had been renamed in my set of changes:

![image](https://user-images.githubusercontent.com/52560012/154574254-5f0fd897-6092-4259-b096-bc8190940c51.png)

I have removed the file added as a result of my renaming so that there is no change introduced relative to the `develop` branch.